indicators/forms.py

```diff
@@ -1,9 +1,29 @@
 from django import forms
+
 from .models import CountryIndicator
+from .validators import validate_identity, validate_whitespaces, \
+    validate_english_letters
 
 
 class CountryIndicatorForm(forms.ModelForm):
 
+    def __init__(self, *args, **kwargs):
+        super(CountryIndicatorForm, self).__init__(*args, **kwargs)
+        self.fields['iso_code'].strip = False
+
     class Meta:
         model = CountryIndicator
         fields = '__all__'
+        help_texts = {
+            'iso_code': ('*Iso code must contains exactly 3 characters<br>'
+                         '*Must contains only latin letters<br>'
+                         '*Must be only letters'),
+            }
+
+    def clean_iso_code(self):
```

Comment:
Seems like you're trying to reinvent the wheel. According to `help_texts` I guess next approach may fits you well:
```python
import re

from django.core.validators import RegexValidator
# other imports here

# pre-compiled regexp in module-level "constant" allows
# us to save some execution time
_RE_ISO_CODE = re.compile(r'^[a-z]{3}$', re.IGNORECASE)

class CountryIndicator(models.Model):
    # some important descriptions
    iso_code = models.CharField(max_length=3, unique=True,                            
                           validators=RegexValidator(_RE_ISO_CODE))
    # another important items
```